stories/test_data/fixtures.ts

```diff
@@ -288,6 +288,14 @@ export const comboWithTwoMeasuresAndViewByAttributeMdObject: VisualizationObject
         .buckets,
 };
 
+export const comboWithThreeMeasuresAndViewByAttribute = barChartWith3MetricsAndViewByAttribute;
+export const comboWithThreeMeasuresAndViewByAttributeMdObject: VisualizationObject.IVisualizationObjectContent = {
+    buckets: require("../test_data/combo_chart_with_three_measures_view_by_attribute_md_object.json").buckets,
+    filters: require("../test_data/combo_chart_with_three_measures_view_by_attribute_md_object.json").buckets,
```

Comment:
'filters' and 'visualizationClass' from this are not used in code (logic is driven only by buckets), but still I would not put there 'buckets' from json file

Reply:
'filters' and 'visualizationClass' are removed